pkg/virt-launcher/virtwrap/converter/converter_test.go

```diff
@@ -2028,6 +2029,99 @@ var _ = Describe("Converter", func() {
 				"expected number of queues to equal number of requested vCPUs")
 		})
 	})
+
+	Context("Correctly handle IsolateEmulatorThread with dedicated cpus", func() {
+		DescribeTable("should succeed assigning CPUs to emulatorThread",
+			func(cores uint32, converterContext *ConverterContext, CPUManagerPolicyBetaOption v1.CPUManagerPolicyBetaOptions,
```

Comment:
@RamLavi I think it worth to check that we don't overlap with assignment here, that the CPUsets for the emulator threads are not used by VCPU pinning for the guest.

Reply:
great idea! 
[DONE](https://github.com/kubevirt/kubevirt/compare/c919c2a4cbb6b8026f1ab23c8e43a2d9acfa0043..fbdee3924ee7e34c214389e3105e791e1b1adf08)